src/images/badges/ctrl_c_ctrl_v.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please set the font color to white to work more consistently with other badges.

![image](https://user-images.githubusercontent.com/8878152/92506714-da3c4d80-f1d3-11ea-935a-067156ae07eb.png)